src/views/users/User/UserList.vue

```diff
@@ -129,11 +129,6 @@ export default {
           ]
         },
         columnsMeta: {
-          name: {
-            formatterArgs: {
-              route: 'UserDetail'
-            }
-          },
           mfa_level: {
             width: '130px',
             formatter: (row) => {
```

Comment:
The difference between the two files is minor but it involves reorganizing some of the elements like `mfaLevel` column's field and renaming a row formatting function (`formatter`). The key points I can see are:

- Column `name` doesn't match the previous one.

Here's what you need to do:
  * Replace `{@}...+{}` with ```... ```
    @@ -129,11 +129,6 @@
           ],
         columnsMeta: {
-          "Name": {
               ...other properties,
-          },
           MFA Level:
...
      },